Ah. was somewhat expecting that linter to trigger 😂 

```
daemon/daemon_unix.go:1633:55: S1002: should omit comparison to bool constant, can be simplified to !val (staticcheck)
	if val, ok := cfg.Features["time-namespaces"]; ok && val == false {
	                                                     ^
```

--- a/daemon/daemon_unix.go
+++ b/daemon/daemon_unix.go
@@ -1623,7 +1623,20 @@ func getSysInfo(cfg *config.Config) *sysinfo.SysInfo {
 			siOpts = append(siOpts, sysinfo.WithCgroup2GroupPath("/user.slice/user-"+euid+".slice"))
 		}
 	}
-	return sysinfo.New(siOpts...)
+	si := sysinfo.New(siOpts...)
+
+	// The "time-namespaces" feature-flag is a (temporary) escape-hatch to disable
+	// the use of time-namespaces for containers. This allows users to return to the
+	// previous default, which unconditionally used the host's time-namespace.
+	//
+	// We can consider removing this flag if there's no regressions; see https://github.com/moby/moby/pull/52326#issuecomment-4401495854
+	if enabled, ok := cfg.Features["time-namespaces"]; ok && !enabled {
+		if si.TimeNamespaces {
+			si.TimeNamespaces = false
+			log.G(context.TODO()).Info("time-namespaces disabled through feature-override")
+		}
+	}
+	return si
 }
 
 func recursiveUnmount(target string) error {

